ui/src/redux/modules/models/updateModel.js

```diff
@@ -37,15 +37,16 @@ const reduceClearPending = (state, { schema, id }) =>
 /**
  * ACTIONS
  */
-const updateModel = ({ schema, id, path, value, silent = false }) =>
-  ({
+const updateModel = ({ schema, id, path, value, silent = false }) => {
+  return ({
```

Comment:
Linter won't be happy about this, probably just revert it (presumably was for logging)